client/src/components/categorical/value.js

```diff
@@ -32,7 +33,11 @@ class CategoryValue extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      editedLabelText: ""
+      editedLabelText: String(
+        props.categoricalSelection[props.metadataField].categoryValues[
```

Comment:
nit - this lookup occurs all over the file, and probably should be a helper function (for example, see the same thing in handleDeleteValue())

Reply:
I think that doing something like I suggested would help the code clarity throughout these files.

Reply:
@colinmegill has elected to leave this for now, and handle in a future refactor.